ios/Video/CustomPlayerController/RCTPlaybackController.swift

```diff
@@ -616,12 +635,43 @@ class RCTPlaybackController: UIView, AVRoutePickerViewDelegate {
         self.curTimeLabel.text = secondsToTimeLabel(seconds)
     }
     
-    func setUi_durationTime(seconds: Float){
-        if(_isLive){
-            self.durTimeLabel.text = "Live"
-        }else{
-            self.durTimeLabel.text = secondsToTimeLabel(seconds)
+    func setTimeLabelSize(isHours: Bool){
```

Comment:
Nit: with functions that start with `set...` you expect the function argument to be the value that you're setting. Going by this name, I would expect the argument to be a `CGSize` or a width and a height. Perhaps a better name would be `refreshTimeLabelSize` / `updateTimeLabelSize`